meta-cube/recipes-support/overc-utils/source/cube-ctl

```diff
@@ -1545,10 +1545,7 @@ IFS=$OLDIFS
 	    echo ${open_container_running} | grep -q ${container_name}
 	    if [ $? -eq 0 ]; then
 		echo "[INFO] stopping container ${container_name}"
-		cube-cmd ${runc_essential} kill ${container_name} KILL
-		if [ "${cmd}" == "stop" ]; then
-		    cube-cmd ${runc_essential} delete ${container_name}
-		fi
+		cube-cmd ${runc_essential} delete -f ${container_name}
```

Comment:
We actually can't do this right here.

The kill == pause, and the commands are combined in the case statement.

We need to split out the stop|pause) part of the case statement. stop == delete, pause == kill.

This functionality always was in runc, was removed, and has been restored. So trusting it to stay consistent is tricky.

Reply:
Perhaps adding the -f to just the delete part will suffice from the original logic.  The problem is that there is no way to restart after a hard kill with the "c3 start cube-desktop" if the delete had failed previously.  The race happens much more frequently with the latest version of runc.